### DISCUSSION TITLE
How do I select Node option instead of Browser when using `npm init @eslint/config`

### DISCUSSION THREAD

**Discussion by qmarquez** [eslint/eslint#17999] [General] [answered]
### Environment

Node version: 20
npm version: 10
Local ESLint version: 8.56
Global ESLint version: -
Operating System: Windows

### What parser are you using?

Default (Espree)

### What did you do?

The issu it's with the command ```npm init @eslint/config``` the 5th step, "Where dos your code run?" no matter which option I choose node or browser the eslintrc.json set browser to true

### What did you expect to happen?

When I set node browser should not be setted

### What actually happened?

When I set node browser set true

### Link to Minimal Reproducible Example

https://stackblitz.com/abcd1234

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by Rec0iL99** [comment, marked as answer]
Hi @qmarquez, did you use `<space>` to select `Node`? My wild guess is that you used `<enter>` to select `Node`. By default `Browser` is selected, you can remove that selection by pressing the `<space>` key on the `Browser` option and using `<space>` key again to select the `Node` option.

![Screenshot 2024-01-16 at 10 26 03 PM](https://github.com/eslint/eslint/assets/56463684/026c095e-c9bd-48ff-b7f5-9aa18cb07e84)

Now pressing the `<enter>` key should select `Node` for you.

**Reply by qmarquez** [reply]
Ohhh okey, i didn't notice it's multi select, like all the previous questinos was single answer. Yes, was a bad of mine
